CoreData/NSManagedObject+S2MAdditions.m

```diff
@@ -0,0 +1,306 @@
+//
+//  NSManagedObject+S2MAdditions.m
+//  S2MToolbox
+//
+//  Created by ParkSanggeon on 13. 10. 22..
+//  Copyright (c) 2013년 SinnerSchrader Mobile. All rights reserved.
+//
+
+#import "NSManagedObject+S2MAdditions.h"
+
+@implementation NSManagedObject (S2MAdditions)
+
++ (NSEntityDescription*)entityInManagedObjectContext:(NSManagedObjectContext*)moc_
+{
+#warning use mogenerator
```

Comment:
This is a bit ugly. We should make sure this works with projects that don't use MOGenerator, or that have a different MOGenerator template file.